packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -153,7 +153,10 @@ export const getClassMembers = (
         })
       );
     } else if (typescript.isPropertyDeclaration(node)) {
-      if (!typescript.isIdentifier(node.name)) {
+      if (
+        !typescript.isIdentifier(node.name) &&
+        !typescript.isPrivateIdentifier(node.name)
+      ) {
```

Comment:
No action needed: thinking out loud. Regarding this diagnostic below, I wonder if it really deserves being a warning. It's not a problem with the code, it's a limitation of the analyzer. I hope we have a way of ignoring errors at the line and project level.